chess.html

```diff
@@ -0,0 +1,29 @@
+<meta charset="utf-8">
+<script type="text/javascript">
+let field = document.createElement('div');
+document.body.appendChild(field);
+
+for(let y=0;y < 10;y++)
+{
+    for(let x=0;x < 10;x++)
+    {
+        let css = x ? {} : {clear:'left'};
+        let content = '';
+        if(y==0 || y==9)
```

Comment:
```suggestion
        if(y === 0 || y === 9)
```
лучше использовать строгое сравнение, а так же не забывайте про оформление кода